examples/json_test/json_test.c

```diff
@@ -0,0 +1,91 @@
+/* Very basic example that just demonstrates we can run at all!
```

Comment:
Scrolling further down I realised this is adapted from the jmsn simple example, so we need to mark its copyright/license accordingly.